objects/src/testing/account_code.rs

```diff
@@ -58,10 +32,17 @@ pub const DEFAULT_AUTH_SCRIPT: &str = "
 impl AccountCode {
     /// Creates a mock [AccountCode] that exposes wallet interface
     pub fn mock_wallet(assembler: Assembler) -> AccountCode {
-        let account_code = "\
+        let library = assembler.assemble_library(&[Self::mock_module()]).unwrap();
+        AccountCode::new(library).unwrap()
+    }
+
+    /// Creates a mock [Module] which can be used to assemble programs and as a library to create a
+    /// mock [AccountCode] interface. Transaction and note scripts that make use of this interface
+    /// should be assembled with this.
+    pub fn mock_module() -> Module {
+        let code = "
         use.miden::account
         use.miden::tx
-
         # acct proc 0
```

Comment:
Do we still need to keep this numbering (i.e., `acct proc 0`, `acct proc 1` etc.)?